include/nlohmann/json.hpp

```diff
@@ -2912,7 +2912,7 @@ class basic_json
     written using `at()`. It also demonstrates the different exceptions that
     can be thrown.,at__size_type}
     */
-    reference at(size_type idx)
+    basic_json& at(size_type idx)
```

Comment:
I think the change from `reference` to `basic_json&` is not improving readability. Unless there are good reasons, I would not approve this PR.